app/konnect/updates.md

```diff
@@ -8,6 +8,41 @@ an application that lets you manage configuration for multiple runtimes
 from a single, cloud-based control plane, and provides a catalog of all deployed
 services.
 
+## November 2021
+### 2021.11.10
+**{{site.base_gateway}} 2.6.0.0 support**
+: {{site.konnect_saas}} now supports {{site.base_gateway}} 2.6.0.0.
+runtimes. You can keep using existing 2.5.x runtimes, or you can upgrade to
+2.6.0.0 to take advantage of any new features, updates, and bug fixes.
+: This release introduces the new [jq plugin](/hub/kong-inc/jq). It also
+adds new features and improvements to a long list of plugins, including:
+* Broader auth support for [Kafka Log](/hub/kong-inc/kafka-log)
+and [Kafka Upstream](/hub/kong-inc/kafka-upstream)
+* The [Prometheus](/hub/kong-inc/prometheus) plugin can now track the
+`data_plane_cluster_cert_expiry_timestamp` metric, letting you keep an eye on the
+status of you data plane certificates
+* A new `trigger` configuration option for the
+[Request Termination](/hub/kong-inc/request-termination) plugin, which tells the
+plugin to activate only on specific headers or query parameters
```

Comment:
Do you want to try for parallel construction here? (not required, and I'm evidently not up to the task of suggesting such a fix ;) )